samples/nrf_desktop/src/events/config_event.h

```diff
@@ -34,20 +34,35 @@ enum config_event_id {
 	CONFIG_EVENT_ID_MAX
 };
 
-
 /** @brief Configuration channel event.
  * Used to change firmware parameters at runtime.
  */
 struct config_event {
 	struct event_header header;
 
+	u16_t recipient;
```

Comment:
do we need recipient here? I think all config events target our board

Reply:
We will at some point add a keyboard. And i would not be surprised if we will use it om dongle as well. I.e to enable pairing

Reply:
Yes... we don't check recipient anywhere where this event is received. I think its better to remove it and restore `store_needed`